database.js

```diff
@@ -0,0 +1,28 @@
+const { Client } = require("pg");
+require('dotenv').config()
+
+const client = new Client({
+  host: process.env.HOST,
+  port: process.env.PORT,
+  user: process.env.USERNAME,
+  password: process.env.PASSWORD,
+  database: process.env.DATABASE 
+});
+
+client.connect().then(() => {
+  createPgAccount("username", "password", "database");
+});
+
+const createPgAccount = async (username, password, database) => {
+  try{
+    const createDatabase = await client.query(`CREATE DATABASE ${database}`)
+    
+    const createUser = await client.query(`CREATE USER ${username} WITH ENCRYPTED password '${password}'`)
+
+    const userDatabasePrivileges = await client.query(`GRANT ALL PRIVILEGES ON DATABASE ${database} To ${username}`)
+
+    await client.end()
```

Comment:
why is your client ending when you've created 1 account? By doing this, you can't run this function more than once.